gateway/testdata/valid.yaml

```
@@ -1,7 +1,5 @@
-basic:
-  username1:
-    ID: "orgid1"
-    password: "password1"
-  username2:
-    ID: "orgid2"
-    password: "password2"
\ No newline at end of file
+server-address: "localhost:8080"
+auth-type: "basic"
+routes:
+  - path: "/api/v1"
+    target: "http://localhost:8081"
```

```suggestion
    target: "distributor"
targets:
  distributor: "http://127.0.0.1:8081"
```

That is if you want to still follow the spec you proposed.
And you would need to validate that distributor exists as target.